backend/api/application_form/serializers.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import serializers
+from .models import ApplicationForm
+
+class ApplicationFormSerializer(serializers.ModelSerializer):
+  class Meta:
+    model = ApplicationForm
+    fields = [
+      'first_name',
+      'last_name',
+      'gender',
+      'nationality',
+      'location',
```

Comment:
Just a heads-up: once we get the schools table up we may also need a field that is a school. Not actionable now though.